kfs-core/src/main/java/edu/arizona/kfs/gl/document/validation/GlobalTransactionEditDetailRule.java

```diff
@@ -0,0 +1,177 @@
+/*

```

Comment:
This is the old license. My position on licenses in the edu.arizona code files is either no license (preferred) or the new license.